python-main.js

```diff
@@ -1079,24 +1079,59 @@ function web_editor(config) {
                 }
             })
             .catch(function(err) {
-                console.log("Error connecting: " + err);
-                $("#flashing-overlay-container").css("display", "flex");
-                $("#flashing-info").addClass('hidden');
-
-                // If micro:bit does not support dapjs
-                if (err.message === "No valid interfaces found."){
-                    $("#flashing-overlay-error").html('<div>' + err + '</div><div>'+ config["translate"]["webusb"]["err"]["update-req"] +'</div><a href="#" onclick="flashErrorClose()">'+ config["translate"]["webusb"]["close"] +'</a>');
-                    return;
-                } else if (err.message === "Unable to claim interface.") {
-                    $("#flashing-overlay-error").html('<div>'+ config["translate"]["webusb"]["err"]["clear-connect"] +'</div><a href="#" onclick="flashErrorClose()">'+ config["translate"]["webusb"]["close"] +'</a>');
-                    return;
-                }
-
-                $("#flashing-overlay-error").html('<div>' + err + '</div><div>'+ config["translate"]["webusb"]["err"]["restart-microbit"] +'</div><a href="#" onclick="flashErrorClose()">'+ config["translate"]["webusb"]["close"] +'</a>');
+                    errorHandler(err);
             });
         }).catch(function(err) {
             console.log("There was an error during connecting: " + err);
```

Comment:
Do we want to do anything with this error? Occurs when there's an error initially connecting. But it's also triggered by cancelling the connection dialog

Reply:
I believe this is no longer applicable, right? We can mark this as resolved in that case.